docker-compose.yml

```diff
@@ -73,20 +74,30 @@ services:
       - openam
       - nicsweb
     ports:
-      - "8081:80"
+      - "8085:80"
     volumes:
       - static:/var/www/html/
 
   openam:
     container_name: nics_openam_1
+<<<<<<< HEAD
```

Comment:
Yeah, it looks like the merge got screwed up. Let's discuss if you are available.